book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -225,7 +225,7 @@ to a parent, if one exists.
 Let's also define some constants for convenience:
 
 ```rust,no_run,noplaypen
-const PADDLE_HEIGHT: f32 = 16.0;
+pub const PADDLE_HEIGHT: f32 = 16.0;
```

Comment:
maybe I'm blind but why pub this?

Reply:
It's required in chapter 3, but I could just add a note there, maybe that's a bit more clear